Description: systemd: Fix ntp-wait.service paths
 - ntpwait is installed to /usr/sbin.
 - ntpd.service is called ntpsec.service in this package
Forwarded: not-needed
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2018-07-21
--- a/etc/ntp-wait.service
+++ b/etc/ntp-wait.service
@@ -1,14 +1,14 @@
 [Unit]
 Description=Wait for ntpd to synchronize system clock
 Documentation=man:ntpwait(8)
-Requisite=ntpd.service
-After=ntpd.service
+Requisite=ntpsec.service
+After=ntpsec.service
 Conflicts=systemd-timesyncd.service
 ConditionCapability=CAP_SYS_TIME
 
 [Service]
 Type=oneshot
-ExecStart=/usr/bin/ntpwait -s 1 -n 30000
+ExecStart=/usr/sbin/ntpwait -s 1 -n 30000
 RemainAfterExit=yes
 StandardOutput=null
 
